site/docs/encoding/condition.md

```diff
@@ -69,9 +84,9 @@ __Note:__ When using a conditional field definition, only a `value` may be speci
 }
 ```
 
-A condition value definition uses a value encoding when data fall within a selection. When data lie outside a selection, either a value or field encoding may be used.
+A condition value definition uses a constant value encoding when data fall within a selection or satisfy a logical predicate.  A field or value definition can be specified as the "else" conditon when the condition is not satified.
```

Comment:
There is no else condition but an else predicate.

Reply:
I don't know what you're saying the "else" case doesn't have a predicate.

Anyway, I can see that this reads confusing.

Reply:
I'd call it else case, predicate, or branch. Also, there is a typo in "conditon".

Reply:
https://github.com/vega/vega-lite/pull/3275 makes some major changes to the schema.